create_spectrogram.py

```diff
@@ -28,12 +45,12 @@ def save_spectrogram(input_wav, plot_path=None, NFFT=256):
         `plot_path`: Path to the output spectrogram file. Default is `input_wav` with .png extension.
         `NFFT`: The number of data points used in each block for the FFT. A power 2 is most efficient.
     Returns:
-        None
+        Path to the spectrogram.
     """
     samplerate, data = wavfile.read(input_wav)
     noverlap = NFFT // 2 if NFFT <= 128 else 128
 
-    title = input_wav.removesuffix(".wav")
+    title = input_wav.replace(".wav", "")
```

Comment:
with try-except you can catch io error.

Reply:
I've changed this part a bit so it now looks like this `title = path.splitext(path.basename(input_wav))[0]` but what io error can happen here?

Reply:
I think I referred to the line above which is reading the file: `wavfile.read`

Reply:
Okay, I see but why would we want to catch here? If we can't read the file I don't see what else we could do.

Reply:
I see that the error gets logged anyway.